delete.ps1

```diff
@@ -95,53 +93,53 @@ try {
     catch {
         if ($_.Exception.Response.StatusCode -eq 404) {
             $outputContext.AuditLogs.Add([PSCustomObject]@{
-                    Action  = "DeleteAccount" # Optionally specify a different action for this audit log
-                    Message = "YsisV2 account for: [$($p.DisplayName)] not found. Possibly already deleted. Skipping action"
+                    Action  = "DeleteAccount"
+                    Message = "Ysis account for [$($person.DisplayName)] could not be found by account reference [$($actionContext.References.Account)] and is possibly already deleted. Skipping action"
                     IsError = $false
-                })          
+                })
             $responseUser = $null
```

Comment:
All other scripts use a throw (differend throw messages in account scripts and permission scripts), recommend making all the scripts handle the "not found" on the same way

Reply:
Valid point, but not for create. If an account is 'not found' in the create script, a new user is created

Reply:
This is resolved as well (only in delete script, na for create)